tests/test_utils.py

```diff
@@ -255,3 +258,52 @@ def test_trim_commas_df(self, local_manifest):
         trimmed_df = df_utils.trim_commas_df(df_with_nans)
 
         assert_frame_equal(trimmed_df, local_manifest)
+
+
+class TestValidateUtils:
+
+    def test_validate_schema(self, schema_explorer):
+
+        se_obj = schema_explorer("tests/data/schema_org_schemas/example.jsonld")
+
+        expected = validate_utils.validate_schema(se_obj.schema)
+        
+        actual = None
+
+        assert expected == actual
```

Comment:
I believe you should be using `assert actual is None` rather than using the `==`. 

Source: https://stackoverflow.com/a/3257957